src/editor/ZoomControls.tsx

```diff
@@ -10,7 +10,7 @@ import {
   ThemeTypings,
 } from "@chakra-ui/react";
 import { useCallback } from "react";
-import { RiZoomInLine, RiZoomOutLine } from "react-icons/ri";
+import { RiAddLine, RiSubtractLine } from "react-icons/ri";
```

Comment:
I hadn't really registered that the icons were so different in the new design. I think it's fine, just amusing that I didn't notice.